scripts/ci/automation_full_test.py

```diff
@@ -27,8 +27,8 @@
 profile = sys.argv[3]
 serial_modules = sys.argv[4].split()
 fix_failure_tests = sys.argv[5].lower() == 'true' if len(sys.argv) >= 6 else False
-working_directory = "/home/vsts/work/1/s"
-azdev_test_result_dir = "/home/vsts/.azdev/env_config/home/vsts/work/1/s/env"
+working_directory = "/mnt/vss/_work/1/s"
```

Comment:
Is there a predefined environment variable for "working_directory" to use instead of hardcoded?

Reply:
There is a variable Pipeline.Workspace has the value   `/home/vsts/work/1`, but we still need to hardcode `/home/vsts`,  `/home/vsts/work` and `/home/vsts/work/1/s` and etc.
Consider it is a one time job, can we keep using the hardcoding?